lib/dai-plugin-mcd/src/AuctionService.js

```diff
@@ -1,8 +1,9 @@
 import Maker from '@makerdao/dai';
 import Auction from './Auction';
+import { ServiceRoles } from './constants';
 
 export default class AuctionService extends Maker.PublicService {
-  constructor(name = 'auction') {
+  constructor(name = ServiceRoles.AUCTION) {
```

Comment:
@tyler17 general rule with using constants: make sure you use them everywhere--the whole point of constants (apart from helping IDE's and linters catch errors) is that if you have to change them, you only have to change them in one place